.github/workflows/e2e-nightly-master.yml

```diff
@@ -34,11 +34,22 @@ jobs:
       - name: Generate testnets
         working-directory: test/e2e
         # When changing -g, also change the matrix groups above
-        run: ./build/generator -g 4 -d networks/nightly
+        run: ./build/generator -g 4 -d networks/nightly/legacy -p legacy
+        run: ./build/generator -g 4 -d networks/nightly/new -p new
+        run: ./build/generator -g 4 -d networks/nightly/split -p split
```

Comment:
Do you think we should worry about adding a combined version yet or just run the legacy and new

Reply:
Let's run them for now!